src/librbd/deep_copy/ObjectCopyRequest.cc

```diff
@@ -58,10 +58,13 @@ ObjectCopyRequest<I>::ObjectCopyRequest(I *src_image_ctx,
 
   m_dst_oid = m_dst_image_ctx->get_object_name(dst_object_number);
 
-  ldout(m_cct, 20) << "dst_oid=" << m_dst_oid << ", "
-                   << "src_snap_id_start=" << m_src_snap_id_start << ", "
-                   << "dst_snap_id_start=" << m_dst_snap_id_start << ", "
-                   << "snap_map=" << m_snap_map << dendl;
+
```

Comment:
No need to add an extra blank line here.

Reply:
Done.